app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -0,0 +1,380 @@
+    <string name="rewind_memory_usage_above_recommended_limit">存储容量超过推荐的限制</string>
+    <string name="interpolation">插值</string>
+    <string name="bitrate">比特率</string>
+    <string name="input_quick_save">快速保存</string>
+    <string name="input_quick_load">快速读取</string>
+
+
+    <string-array name="fast_forward_speed_multiplier_options" tools:ignore="InconsistentArrays">
+        <item>无限制</item>
+        <item>1.5x</item>
+        <item>2x</item>
+        <item>3x</item>
+        <item>4x</item>
+        <item>8x</item>
+    </string-array>
+
+    <string-array name="rom_icon_filtering_options">
+        <item>无 (像素化)</item>
+        <item>Linear (线性流畅)</item>
+    </string-array>
+
+    <string-array name="game_runtime_console_type_options">
+        <item>默认</item>
+        <item>@string/console_ds</item>
+        <item>@string/console_dsi</item>
+    </string-array>
+
+    <string-array name="firmware_settings_language_options">
+        <item>英文</item>
+        <item>法语</item>
+        <item>德语</item>
+        <item>意大利语</item>
+        <item>西班牙语</item>
+        <item>日语</item>
+    </string-array>
+
+    <string-array name="game_runtime_mic_source_options">
+        <item>默认</item>
+        <item>无</item>
+        <item>吹气</item>
+        <item>系统麦克风</item>
+    </string-array>
+
+    <string-array name="console_type_options">
+        <item>@string/console_ds</item>
+        <item>@string/console_dsi</item>
+    </string-array>
+
+    <string-array name="fps_counter_position_options">
+        <item>隐藏</item>
+        <item>左上</item>
+        <item>顶部居中</item>
+        <item>右上</item>
+        <item>左下</item>
+        <item>底部居中</item>
+        <item>右下</item>
+    </string-array>
+
+    <string-array name="video_filtering_options">
+        <item>无</item>
+        <item>Linear</item>
+        <item>2xBR</item>
+        <item>HQ2X</item>
+        <item>HQ4X</item>
+        <item>Quilez</item>
+        <item>LCD</item>
+    </string-array>
+
+    <string-array name="audio_interpolation_options">
+        <item>无</item>
+        <item>Linear</item>
+        <item>Cosine</item>
+        <item>Cubic</item>
+    </string-array>
+
+    <string-array name="audio_bitrate_options">
+        <item>自动</item>
+        <item>10-bit</item>
+        <item>16-bit</item>
+    </string-array>
+
+    <string-array name="audio_latency_options">
+        <item>低 (较慢)</item>
+        <item>中</item>
+        <item>高 (较快)</item>
+    </string-array>
+
+    <string-array name="mic_source_options">
+        <item>无</item>
+        <item>吹气</item>
+        <item>系统麦克风</item>
+    </string-array>
+
+    <string-array name="save_state_locations">
+        <item>存档文件目录</item>
+        <item>ROM目录</item>
+        <item>内部目录</item>
+    </string-array>
+
+    <string-array name="layout_orientation_options">
+        <item>跟随系统</item>
+        <item>竖向</item>
+        <item>横向</item>
+    </string-array>
+
+    <string-array name="background_portrait_mode_options">
```

Comment:
Missing translations.

Reply:
I can't build this project, so I don't know some string context. If I can run this application, I will do better.